test/dataframes.jl

```diff
@@ -147,8 +178,8 @@ end
     @test @transform!(df, n = :i .* :g).n == [1, 2, 3, 8, 10]
 
     # non-copying
-    @test @transform!(df, n = :i).g === df.g
-    @test @transform!(df, n = :i).n === df.i
+    @test @transform!(df, n = :i).g == df.g
+    @test @transform!(df, n = :i).n == df.i
```

Comment:
This change defeats the purpose of this test, right? :-)

Reply:
It's an upstream change in DataFrames. I should do it in it's own PR.